src/Main.elm

```diff
@@ -95,13 +102,13 @@ update msg model =
                         Array.set index newContent model.paragraphs
             in
             ( { model | paragraphs = newParagraphs }
-            , Cmd.none
+            , Ext.Browser.setPageTitle ("Paragraphs: " ++ String.fromInt (Array.length model.paragraphs))
             )
 
 
-view : Model -> Browser.Document Msg
+view : Model -> Html Msg
 view model =
-    Browser.Document ("Paragraphs: " ++ String.fromInt (Array.length model.paragraphs))
+    div []
```

Comment:
since our view can't include page title anymore, app will need to decide when/how to trigger the `Ext.Browser.setPageTitle` cmd